app/src/main/java/com/snatik/matches/themes/Themes.java

```diff
@@ -1,11 +1,11 @@
 package com.snatik.matches.themes;
 
-import java.util.ArrayList;
-
 import android.graphics.Bitmap;
 
-import com.snatik.matches.common.Shared;
-import com.snatik.matches.utils.Utils;
+import com.obosapps.matches.common.Shared;
```

Comment:
please change all imports back to `com.snatik.matches`. Otherwise it won't compile.